.github/workflows/run-checker-ci.yml

```diff
@@ -23,6 +23,7 @@ jobs:
           no-dtls,
           no-ec,
           no-ec2m,
+          no-http,
           no-siv,
           no-legacy,
           no-rfc3779,
```

Comment:
To compensate for this added build, please remove the no-rfc3779 build - it is already in the run-checker-daily run anyway.

Reply:
okay, done.